app/src/main/java/com/example/mealsnap_prototype_1/LoginActivity.java

```diff
@@ -29,6 +32,16 @@ protected void onCreate(Bundle savedInstanceState) {
         etUsername = findViewById(R.id.etUsername);
         etPassword = findViewById(R.id.etPassword);
         btnLogin = findViewById(R.id.btnLogin);
+        signUp = findViewById(R.id.signUp);
+
+        signUp.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                Log.i(TAG, "onClick Sign Up Button");
+                Toast.makeText(LoginActivity.this, "Moving to Sign Up", Toast.LENGTH_SHORT).show();
+                goToSignupActivity();
```

Comment:
Can you instead create a new activity called `OnboardingActivity`, which would have the signup and login buttons and will handle these navigations?